traffic_portal/app/src/traffic_portal_properties.json

```diff
@@ -13,6 +13,9 @@
       "currentStats": {
         "refreshRateInMS": 30000
       },
+      "deliveryServiceGbps": {
+        "refreshRateInMS": 30000
```

Comment:
let's make this a minute. i don't think the data is any more granular that a minute so making 2 calls a minute doesn't make sense really.